pkg/crc/machine/kubeconfig.go

```diff
@@ -174,6 +174,15 @@ func addContext(cfg *api.Config, ip string, clusterConfig *types.ClusterConfig,
 	if err != nil {
 		return err
 	}
+
+	// append /clustername to AuthInfo
+	url, err := url.Parse(clusterConfig.ClusterAPI)
+	if err != nil {
+		return err
+	}
+	clusterURL := url.Host
+	username = fmt.Sprintf("%s/%s", username, clusterURL)
```

Comment:
Not sure why the `clusterURL` variable is used as it does not add much more clarity
```golang
username = fmt.Sprintf("%s/%s", username, url.Host)
```